src/test/java/HomeWork18.java

```diff
@@ -0,0 +1,41 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class HomeWork18 extends BaseTest {
+
+    @Test
+    public void playSong() throws InterruptedException {
+
+
+        navigateToPage();
+        provideEmail("karina.usmanova01@testpro.io");
+        providePassword("YrEdlRVe");
+        clickLoginButton();
+        Thread.sleep(2000);
+
+        clickPlayNextSongButton();
+        Thread.sleep(2000);
+        clickPlayButton();
+        Thread.sleep(2000);
+        showPauseBtn();
+        Thread.sleep(2000);
+        boolean verifyPauseBtn = driver.findElement(By.xpath("//div[@class='side player-controls']//i[@class='fa fa-pause']")).isDisplayed();
+    }
```

Comment:
**Use in test methods, the following logical split:   GIVEN – WHEN - THEN**
    
For example

```
 @Test

    public void testChangeProfileName (){
        // GIVEN

        String newName  = getRandomString();

        String currentPassword = "te$t$tudent";

        provideEmail("demo@class.com");

        providePassword();

        clickSubmitBtn();

 

        // WHEN

        openUserProfilePage();

        setName(newName);

        setCurrentPassword(currentPassword);

 

 

        // THEN

        Assert.assertTrue(getSuccessPopUp().isDisplayed());

}
```

Please, add Assert

**One test - One Assert**

For example

![18](https://github.com/Test-Pro-Bootcamp/A57-QA-Automation-Framework/assets/104761482/a5c839f7-3cd5-44b5-89e4-749dcf29d29b)